mllib/src/main/scala/org/apache/spark/ml/tree/treeModels.scala

```diff
@@ -212,15 +212,13 @@ private[ml] object TreeEnsembleModel {
   def computeFeatureImportance(
       node: Node,
       importances: OpenHashMap[Int, Double]): Unit = {
-    node match {
-      case n: InternalNode =>
-        val feature = n.split.featureIndex
-        val scaledGain = n.gain * n.impurityStats.count
-        importances.changeValue(feature, scaledGain, _ + scaledGain)
-        computeFeatureImportance(n.leftChild, importances)
-        computeFeatureImportance(n.rightChild, importances)
-      case n: LeafNode =>
-      // do nothing
+    if (node.isInstanceOf[InternalNode]) {
```

Comment:
Why change this?  It may be better to use match-case since that will give warnings if we ever add new types of nodes.